### PR TITLE
Updated SwiftPrivate.swift using String Interpolation

### PR DESCRIPTION
**Pull request by arguiot**
I updated /stdlib/private/SwiftPrivate/SwiftPrivate.swift using String Interpolations, so the code is more readable and more "Swifty". I also updated the copyright to 2018.

This PR doesn't resolve any bugs or issue.

### CODE REVIEW

**Comment by arguiot** [comment]
@swift-ci Please smoke test

File: stdlib/private/SwiftPrivate/SwiftPrivate.swift [hunk 1]
```diff
@@ -13,69 +13,69 @@
...
 }
 
 /// Convert the given sequence of numeric values to a string representing
 /// their hexadecimal values.
-  // FIXME(integers): support a more general BinaryInteger protocol
+// FIXME(integers): support a more general BinaryInteger protocol
 public func asHex<S : Sequence>(_ x: S) -> String
-  where
-  S.Element : FixedWidthInteger {
-  return "[ " + x.lazy.map { asHex($0) }.joined(separator: ", ") + " ]"
```
Comment (harlanhaskins) [inline stdlib/private/SwiftPrivate/SwiftPrivate.swift]: Can you change these indentations back to 2 spaces? The Swift standard library prefers 2-space indentation.

**Review by lorentey — CHANGES_REQUESTED** [review]
We prefer to use 2 spaces for indentation; otherwise the change looks good to me!

**Comment by arguiot** [comment]
@xwu I applied your changes

**Comment by moiseev** [comment]
@swift-ci Please smoke test

**Review by moiseev — APPROVED** [review]
:shipit:

**Review by lorentey — APPROVED** [review]
Looking good!

**Pull request merged**